### PR TITLE
Use consistent user plugin order

### PR DESCRIPTION
**Pull request by marvinhagemeister**

### CODE REVIEW

**Review by JoviDeCroock — APPROVED** [review]

File: packages/wmr/src/bundler.js [hunk 1]
```diff
@@ -87,14 +87,16 @@ export async function bundleProd({
 		input.push('./' + pathToPosix(relative(root, abs)));
 	});
 
+	// Plugins are pre-sorted
+	const split = plugins.findIndex(p => p.enforce !== 'post');
```
Comment (marvinhagemeister) [inline packages/wmr/src/bundler.js:91]: Wait this doesn't make sense. The condition should be inverted

Comment (developit) [reply packages/wmr/src/bundler.js:91]: It's supposed to be `!== 'pre'` - the index that marks the end of the "pre" items.

Comment (developit) [reply packages/wmr/src/bundler.js:91]: Basically the first index of null/'normal'/'post'.

Comment (marvinhagemeister) [reply packages/wmr/src/bundler.js:91]: Imo `normal` plugins should be moved to the start too. You'll usually want to intercept stuff before we'd internal plugins. It was one of the key learnings in vite

**Pull request merged**